src/hypergraph.jl

```diff
@@ -295,4 +295,39 @@ function nhv(h::Hypergraph{T, V, E}) where {T <: Real, V, E}
     length(h.v2he)
 end
 
+"""
+    _walk!(h::Hypergraph, s::AbstractVector{Int}, i::Int, visited::AbstractVector{Bool})
+
+Return the list of neighbors of a given vertex (an auxiliary function for `get_connected_components`).
```

Comment:
It does not return anything. The description of function should be something like this (to be checked):
Appends the list of neighbors `s` for a given vertex `i` (an auxiliary function for `get_connected_components`).